payments-core/src/main/java/com/stripe/android/cards/DefaultStaticCardAccountRanges.kt

```diff
@@ -176,6 +176,31 @@ class DefaultStaticCardAccountRanges : StaticCardAccountRanges {
             )
         }
 
+        private val CARTES_BANCAIRES_ACCOUNT_RANGES = setOf(
+            BinRange(
+                low = "2221000000000000",
+                high = "2720000000000000"
+            ),
+            BinRange(
+                low = "4000000000000000",
+                high = "4999999999999999"
+            ),
+            BinRange(
+                low = "5000000000000000",
+                high = "5900000000000000"
+            ),
+            BinRange(
+                low = "6700000000000000",
+                high = "6799999999999999"
+            )
+        ).map {
+            AccountRange(
+                binRange = it,
+                panLength = 16,
+                brandInfo = AccountRange.BrandInfo.CartesBancaires
+            )
+        }
```

Comment:
How do these bin ranges impact other cards besides the test cards? Is there any reason why we can't hardcode the  test numbers rather than these ranges? In production the card metadata service determines what card brands to vend to the client. I'm concerned these bin ranges could have unintended consequences. 

https://github.com/stripe/stripe-ios/blob/ac4a8468/StripePayments/StripePayments/Source/Helpers/STPCardValidator.swift#L444
https://github.com/stripe/stripe-android/blob/5e021d15/payments-core/src/main/java/com/stripe/android/cards/CbcTestCardDelegate.kt#L10

Reply:
I do see failing tests relating to CBC and BIN in the bitrise run, I think we should hardcode these card numbers rather than have the test cards influence client side bin ranges.

Reply:
Looks like StripeJS also hardcodes: https://git.corp.stripe.com/stripe-internal/stripe-js-v3/blob/4900dd4d/src/elements/inner/card/Bins.ts#L78

Reply:
Sure, I can hardcode the test numbers. That seems good so that we can de-risk getting this to Deliveroo and to ensure we don't introduce any unintended behavior.

The BIN ranges that I added here match [the regex](https://github.com/stripe/stripe-android/blob/9f85aa7563c8bb82b3cb4333b73b2ce7ec6a7d7e/payments-model/src/main/java/com/stripe/android/model/CardBrand.kt#L159-L175) that we use for CBC. This API and CBC don't actually use the same code path for determining possible brands. Created [a jira](https://jira.corp.stripe.com/browse/RUN_MOBILESDK-3094) to unify the two code paths because it seems less than ideal that this API would have different behavior than our implementation of CBC.

Reply:
Updated PR to just add the test numbers and no other ranges